### ISSUE TITLE
v5 pulling in v4 module as well

### ISSUE THREAD

**Issue by joescharf** [didip/tollbooth#82]
Using the "Five Minute Tutorial" example from the main page of this repo, (removing the `time` import), and referencing the v5 version of this module also imports the v4.0.2 version of tollbooth as well:

from go.mod:
```
github.com/didip/tollbooth v4.0.2+incompatible // indirect
github.com/didip/tollbooth/v5 v5.1.0
```

I came about this issue when trying to implement `toolbooth_gorestful` and was getting type mismatch errors that differed as a result of the v5 tag:

```
cannot use lmt (type *"github.com/didip/tollbooth/limiter".Limiter) as type *"github.com/didip/tollbooth/v5/limiter".Limiter in argument to RestfulLimitHandler
```

It seems the 5.1.0 release package includes `tollbooth.go` without the /v5 in the import paths:
```
	"github.com/didip/tollbooth/errors"
	"github.com/didip/tollbooth/libstring"
	"github.com/didip/tollbooth/limiter"
```

**Comment by mpe85** [comment]
@didip I saw this is already fixed in the master branch. Will we see another release soon containing that fix?

**Comment by didip** [comment]
Published tag 5.1.1

**Comment by mpe85** [comment]
Works for me 👍
